pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -81,8 +84,28 @@ def work(self):
                                     pokeball = 2 # then use great balls
                                 elif balls_stock[3] > 0: # or if great balls are out of stock too, and player has ultra balls...
                                     pokeball = 3 # then use ultra balls
+                            
+                            ## Use berry to increase success chance.
+                            berry_id = 701 # @ TODO: use better berries if possible
+                            berries_count = self.bot.item_inventory_count(berry_id)
+                            if(catch_rate[pokeball-1] < 0.5 and berries_count > 0): # and potion is in stock
+                                success_percentage = '{0:.2f}'.format(catch_rate[pokeball-1]*100)
+                                logger.log('[x] Catch Rate with normal Pokeball is low ({}%). Throwing {}... ({} left!)'.format(success_percentage,self.item_list[str(berry_id)],berries_count-1))
+                                self.api.use_item_capture(
+                                    item_id=berry_id, 
+                                    encounter_id = encounter_id, 
+                                    spawn_point_guid = spawnpoint_id
+                                )
+                                response_dict = self.api.call()
+                                if response_dict and response_dict['status_code'] is 1:
+                                
+                                    for i in range(len(catch_rate)):
+                                        catch_rate[i] = catch_rate[i] * response_dict['responses']['USE_ITEM_CAPTURE']['item_capture_mult']
```

Comment:
Been testing this for a while and at times, item_capture_mult doesn't exist in USE_ITEM_CAPTURE, so it throws an error and crashes the bot.
Should add a if statement to prevent it from happen.